backend/src/app.ts

```diff
@@ -127,10 +127,8 @@ app.post('/new-landlord', async (req, res) => {
 });
 
 const isLandlord = (obj: LandlordWithId | ApartmentWithId): boolean => 'contact' in obj;
-
-app.get('/search', async (req, res) => {
+app.get('/getAllData', async (req, res) => {
```

Comment:
Sorry I didn't think about this earlier, but what do you think about changing this to a `POST` request? Technically we are just asking the backend to do something, and the frontend is not "getting" any data
```
app.post('/set-data', async (req, res) => {...})
```

Reply:
Also it is best practice in API design for routes to use `-` instead of `camelCase`, e.g. `/set-data` instead of `/setData`! Just a small thing